rotkehlchen/assets/spam_assets.py

```diff
@@ -0,0 +1,26 @@
+from rotkehlchen.constants.resolver import strethaddress_to_identifier
+
+KNOWN_ETH_SPAM_TOKENS = [
```

Comment:
Maybe at some point we need to think to have those defined elsewhere though. In a file outside of the repo. Or even read a DB that has lots of those scam tokens. Doesn't cryptoscamdb do that? https://documenter.getpostman.com/view/4298426/RzZ7nKcM#ed5ac634-fb3f-4f96-8256-b709e4a896fc

For example this call: https://api.cryptoscamdb.org/v1/addresses

I am thinking of what the right long-term approach is here.

Reply:
I didn't know about this page. If there is a list already we can use it and contribute if needed. I'm thinking in downloading it and completing this list ourselves to minimize the number of requests to external resources. Can be done in this PR if you are okey. Also we can just query that endpoint, process the entries and return a list but we have to tkae care of remote errors

Reply:
Hey @409h we want to use cryptoscamdb instead of keeping our own list for scam tokens. Since we now scan all transactions and detect all tokens our users can get lots of illegitimate tokens in their balance/history which should just be ignored.

Users have their own ignore list, but we want (in this PR) to add lots of scam tokens by default to it. And cryptoscamdb sounds like the best choice.

@yabirgb 

> I'm thinking in downloading it and completing this list ourselves to minimize the number of requests to external resources. 

Wouldn't that create a huge file? Why duplicate the data? Where would we put it? Certainly not in the app itself as that would just further increase the size.

> Also we can just query that endpoint, process the entries and return a list but we have to tkae care of remote errors

This sounds like the right approach. Remote error would mean bail and do nothing.

The problem with remote queries is that they can take long and are not good for a DB upgrade though. So we would need to think when it would happen.

We still have the data migrations which are for exactly that purpose (soft DB changes, happening after login) so perhaps use that?